app/api.py

```diff
@@ -371,23 +372,22 @@ async def feedback_read(mentor_id: Optional[str] = None, ticket_id: Optional[str
 
 
 @API.post("/create/Feedback")
-async def feedback_create(mentee_id: str, mentor_id: str, feedback: str):
+async def feedback_create(data: Feedback):
```

Comment:
Regarding my previous comment. `Feedback` the class would be caps still. But the URL txt in line 374 should be lower. Unless I'm missing something.

Reply:
We are taking this out and moving it to a separate PR for the API. Logan is handling the bulk of that.